### PR TITLE
Code style refactoring

### PR DESCRIPTION
**Pull request by azevedo-252**

### CODE REVIEW

File: app/js/books_page/books_page_ctrl.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 'use strict';
 
-class BooksPageCtrl {
-  constructor() {
-    this.searchText = '';
-  }
+function BooksPageCtrl() {
+  let vm = this;
```
Comment (gabrielpoca) [inline app/js/books_page/books_page_ctrl.js]: why not call it `ctrl`? What's the meaning of `vm`?

Comment (azevedo-252) [inline app/js/books_page/books_page_ctrl.js]: ViewModel
It's more of a view model than a conventional controller.

But now that I think about it, I'm already using "ctrl" in the views. So i should make it more consistent.
Which do you prefer? We're more accustomed to controller, but Angular's "controller" acts as a view model

Comment (gabrielpoca) [inline app/js/books_page/books_page_ctrl.js]: I would go with `ctrl`. It's easier to understand, it matches with what it is and we are already using it in other projects.

Comment (azevedo-252) [inline app/js/books_page/books_page_ctrl.js]: Ok i can go with it

**Comment by azevedo-252** [comment]
@gabrielpoca bump

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**